modules/bogo/includes/BogoDataWrapper.php

```
@@ -0,0 +1,92 @@
+	 */
+	public function get_item( $id ) {
+		return BogoDataManager::get_bogo_offer( $id );
+	}
+
+	/**
+	 * Get BOGO offers with pagination support.
+	 *
+	 * @param array $args Query arguments.
+	 * @return array Array with data, pagination info, etc.
+	 */
+	public function get_items( $args = array() ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)